redisinsight/ui/src/pages/slowLog/components/SlowLogConfig/SlowLogConfig.tsx

```diff
@@ -0,0 +1,227 @@
+  const { loading, durationUnit: durationUnitStore } = useSelector(slowLogSelector)
+  const {
+    slowlogMaxLen = DEFAULT_SLOWLOG_MAX_LEN,
+    slowlogLogSlowerThan = DEFAULT_SLOWLOG_SLOWER_THAN,
+  } = useSelector(slowLogConfigSelector)
+
+  const [maxLen, setMaxLen] = useState(`${slowlogMaxLen}`)
+  const [slowerThan, setSlowerThan] = useState(validateNumber(`${slowlogLogSlowerThan}`, Infinity, -1))
+  const [durationUnit, setDurationUnit] = useState(durationUnitStore ?? DEFAULT_SLOWLOG_DURATION_UNIT)
+
+  const dispatch = useDispatch()
+
+  const onChangeUnit = (value: DurationUnits) => {
+    setDurationUnit(value)
+  }
+
+  const handleDefault = () => {
+    setMaxLen(`${slowlogMaxLen}`)
+    setSlowerThan(`${slowlogLogSlowerThan}`)
+    setDurationUnit(durationUnitStore ?? DEFAULT_SLOWLOG_DURATION_UNIT)
+  }
+
+  const handleCancel = () => {
+    closePopover()
+  }
+
+  const handleSave = () => {
+    dispatch(patchSlowLogConfigAction(
+      instanceId,
+      {
+        slowlogMaxLen: +maxLen,
+        slowlogLogSlowerThan: convertNumberByUnits(+slowerThan || -1, durationUnit)
+      },
+      durationUnit,
+      onSuccess
+    ))
+  }
+
+  const onSuccess = () => {
+    setDBConfigStorageField(instanceId, ConfigDBStorageItem.slowLogDurationUnit, durationUnit)
+
+    onRefresh()
+    closePopover()
+  }
+
+  const handleGoWorkbenchPage = (e: React.MouseEvent) => {
+    e.preventDefault()
+    history.push(Pages.workbench(instanceId))
+  }
+  const handleOpenCli = (e: React.MouseEvent) => {
+    e.preventDefault()
+    dispatch(openCli())
+  }
+
+  const clusterContent = () => (
+    <>
+      <EuiText color="subdued" className={styles.clusterText}>
+        {'Each node can have different Slow Log configuration in a clustered database. Use '}
+        <a
+          tabIndex={0}
+          onClick={handleOpenCli}
+          className={styles.link}
+          data-testid="internal-cli-link"
+          onKeyDown={() => ({})}
+          role="link"
+          rel="noreferrer"
+        >
+          CLI
+        </a>
+        {' or '}
+        <a
+          tabIndex={0}
+          onClick={handleGoWorkbenchPage}
+          className={styles.link}
+          data-testid="internal-workbench-link"
+          onKeyDown={() => ({})}
+          role="link"
+          rel="noreferrer"
+        >
+          Workbench
+        </a>
+        {' to configure it.'}
+      </EuiText>
+
+      <EuiSpacer size="xs" />
+      <EuiButton
+        fill
+        color="secondary"
+        className={styles.clusterBtn}
+        onClick={closePopover}
+        data-testid="slowlog-config-ok-btn"
+      >
+        Ok
+      </EuiButton>
+    </>
+  )
+
+  return (
+    <div className={cx(styles.container, { [styles.containerCluster]: connectionType === ConnectionType.Cluster })}>
+      { connectionType === ConnectionType.Cluster && (clusterContent())}
+      { connectionType !== ConnectionType.Cluster && (
```

Comment:
no need space here, and for the line above

Reply:
removed